packages/gatsby-theme-docs/src/components/notifications.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+
+import UnstyledInformationIcon from '../icons/information-icon.svg';
+import UnstyledWarningIcon from '../icons/warning.svg';
+import UnstyledErrorIcon from '../icons/error.svg';
+import createStyledIcon from '../utils/create-styled-icon';
+import markdown2React from '../utils/markdown-2-react';
+import { pxToRem, colors, tokens, dimensions } from '../design-system';
+
+const Container = styled.div`
+  display: flex;
+  width: 100%;
+  padding: ${dimensions.spacings.m};
+  margin: ${dimensions.spacings.s} 0;
+  border: ${pxToRem('1px')} solid ${colors.light.borderInfo};
+  border-radius: ${tokens.borderRadius6};
+`;
+
+const IconWrapper = styled.div`
+  flex-shrink: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: ${pxToRem('24px')};
```

Comment:
It would make sense to define icon sizes in the design-system, then use those variables (so no direct use of `pxToRem`)